mod/01_filter/set/abstract/filter.rb

```diff
@@ -213,13 +308,18 @@ def multiselect_filter type_codename, label=nil
       names = m.to_name.parts
       # score metric?
       names.length == 3 ? names[2] : names[0]
-    end.uniq!
+    end.uniq!(&:downcase).sort_by!(&:downcase)
     simple_select_filter "designer", [["--", ""]] + designers,
                          Env.params[:designer]
   end
 
+  view :importance_formgroup do
+    options = ["Not Voted", "Upvoted", "Downvoted"]
```

Comment:
I don't this wording is in common use yet, and it may not be clear whose votes are being described.  Perhaps:

* I voted FOR
* I voted AGAINST
* I did NOT vote